Makefile

```
@@ -120,9 +123,44 @@
 	@echo "Checking code with Ruff..."
 	poetry run ruff check src/ tests/
 
-test: ## Run the test suite
-	@echo "Running test suite..."
-	@echo "Linking .env.test to .env for test session..."
+test: ## Run the unit and integration test suite in a clean, isolated environment.
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (12).

(maxbodylength)